controller/testController.go

```
@@ -47,7 +51,13 @@ func (c *TestController) RegisterEndpoints(api *gin.RouterGroup) {
 // @Success 200
 // @Router /test/ [get]
 func (c *TestController) test(ctx *gin.Context) {
-	ctx.JSON(http.StatusOK, "Bokic")
+	tmodel, err := c.testService.ReadAll()
+	if err != nil {
+		ctx.AbortWithError(http.StatusInternalServerError, err)
```

golangci-lint (1.64.8)

Error return value of `ctx.AbortWithError` is not checked

(errcheck)